tests/sim_archive/test_make_snapshot.py

```diff
@@ -2,9 +2,12 @@
 import unittest
 
 from rubin_scheduler.scheduler.schedulers.core_scheduler import CoreScheduler
-from rubin_scheduler.sim_archive import get_scheduler_instance_from_repo
 
-HAVE_TS = importlib.util.find_spec("lsst.ts") is not None
+try:
+    HAVE_TS = importlib.util.find_spec("lsst.ts") is not None
+    from rubin_scheduler.sim_archive import get_scheduler_instance_from_repo
```

Comment:
I've gotten style complaints (from ruff, I think) about using try/excepts to detect whether we have a module, with the suggestion to use find_spec instead.
So, I think what we need is something like
```
HAVE_TS = importlib.util.find_spec("lsst.ts") is not None
if HAVE_TS:
    from rubin_scheduler.sim_archive import get_scheduler_instance_from_repo
```
Yes, I think the try/except approach is more intuitive.

Reply:
I think that only works if there is an `lsst` module. But that's easy enough to check first and still get rid of the try/except.